covalent/_dispatcher_plugins/local.py

```diff
@@ -0,0 +1,72 @@
+# Copyright 2021 Agnostiq Inc.
+#
+# This file is part of Covalent.
+#
+# Licensed under the GNU Affero General Public License 3.0 (the "License").
+# A copy of the License may be obtained with this software package or at
+#
+#      https://www.gnu.org/licenses/agpl-3.0.en.html
+#
+# Use of this file is prohibited except in compliance with the License. Any
+# modifications or derivative works of this file must retain this copyright
+# notice, and modified files must contain a notice indicating that they have
+# been altered from the originals.
+#
+# Covalent is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE. See the License for more details.
+#
+# Relief from the License may be granted by purchasing a commercial license.
+
+import inspect
+from copy import deepcopy
+from functools import wraps
+
+import cloudpickle as pickle
+import requests
+
+from .._results_manager.result import Result
+from .._results_manager.results_manager import get_result
+from .._workflow.lattice import Lattice
+from .base import BaseDispatcher
+
+
+class LocalDispatcher(BaseDispatcher):
+    @staticmethod
+    def dispatch(orig_lattice: Lattice) -> str:
```

Comment:
We would ned docstrings here for it to show up in API part. And also the the dispatcher properties like port etc.. ?

Reply:
Added the dispatcher address and docstrings.